app/script/conversation/ConversationRepository.coffee

```diff
@@ -715,7 +755,7 @@ class z.conversation.ConversationRepository
       generic_message = new z.proto.GenericMessage z.util.create_random_uuid()
       generic_message.set 'lastRead', message_content
 
-      @_send_generic_message @self_conversation().id, generic_message
+      @_add_to_sending_queue @self_conversation().id, generic_message
```

Comment:
I meant in programming.. not in real life :)
https://en.wikipedia.org/wiki/Queue_(abstract_data_type)#Queues_and_programming_languages